flink-examples/flink-examples-streaming/src/main/java/org/apache/flink/streaming/examples/windowing/WindowWordCount.java

```diff
@@ -88,6 +93,12 @@ public static void main(String[] args) throws Exception {
         // available in the Flink UI.
         env.getConfig().setGlobalJobParameters(params);
 
+        if (params.isAsyncState()) {
+            Configuration config = Configuration.fromMap(env.getConfiguration().toMap());
+            config.set(StateBackendOptions.STATE_BACKEND, FORST_STATE_BACKEND_NAME);
+            env.configure(config);
```

Comment:
```suggestion
             if (!config.containsKey(StateBackendOptions.STATE_BACKEND.key())) {
                config.set(StateBackendOptions.STATE_BACKEND, FORST_STATE_BACKEND_NAME);
                env.configure(config);
            }
```
Same for the one in WordCount.java

Reply:
Resolved, After change:
```
        // For async state, by default we will use the forst state backend.
        if (params.isAsyncState()) {
            Configuration config = Configuration.fromMap(env.getConfiguration().toMap());
            if (!config.containsKey(StateBackendOptions.STATE_BACKEND.key())) {
                config.set(StateBackendOptions.STATE_BACKEND, FORST_STATE_BACKEND_NAME);
                env.configure(config);
            }
        }
```